sandbox-backend/src/main/resources/db/migration/V0000034__addUnique.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE discount ADD UNIQUE (name);
```

Comment:
@MaryGoroshko please review this file

Reply:
@akarzhaneuski @AlexSv-j    I've already run the same script in DB, so this one is no longer needed.

Reply:
@AlexSv-j But I missed adding UNIQUE for the address, it would be great if you could add it because now we have duplicate addresses in the address table.